apis/v1alpha1/db_cluster.go

```diff
@@ -0,0 +1,502 @@
+	//
+	// If you are using an AWS SDK tool or the AWS CLI, you can specify SourceRegion
+	// (or --source-region for the AWS CLI) instead of specifying PreSignedUrl manually.
+	// Specifying SourceRegion autogenerates a pre-signed URL that is a valid request
+	// for the operation that can be executed in the source AWS Region.
+	PreSignedURL *string `json:"preSignedURL,omitempty"`
+	// The daily time range during which automated backups are created if automated
+	// backups are enabled using the BackupRetentionPeriod parameter.
+	//
+	// The default is a 30-minute window selected at random from an 8-hour block
+	// of time for each AWS Region. To see the time blocks available, see Adjusting
+	// the Preferred DB Cluster Maintenance Window (https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/USER_UpgradeDBInstance.Maintenance.html#AdjustingTheMaintenanceWindow.Aurora)
+	// in the Amazon Aurora User Guide.
+	//
+	// Constraints:
+	//
+	//    * Must be in the format hh24:mi-hh24:mi.
+	//
+	//    * Must be in Universal Coordinated Time (UTC).
+	//
+	//    * Must not conflict with the preferred maintenance window.
+	//
+	//    * Must be at least 30 minutes.
+	PreferredBackupWindow *string `json:"preferredBackupWindow,omitempty"`
+	// The weekly time range during which system maintenance can occur, in Universal
+	// Coordinated Time (UTC).
+	//
+	// Format: ddd:hh24:mi-ddd:hh24:mi
+	//
+	// The default is a 30-minute window selected at random from an 8-hour block
+	// of time for each AWS Region, occurring on a random day of the week. To see
+	// the time blocks available, see Adjusting the Preferred DB Cluster Maintenance
+	// Window (https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/USER_UpgradeDBInstance.Maintenance.html#AdjustingTheMaintenanceWindow.Aurora)
+	// in the Amazon Aurora User Guide.
+	//
+	// Valid Days: Mon, Tue, Wed, Thu, Fri, Sat, Sun.
+	//
+	// Constraints: Minimum 30-minute window.
+	PreferredMaintenanceWindow *string `json:"preferredMaintenanceWindow,omitempty"`
+	// The Amazon Resource Name (ARN) of the source DB instance or DB cluster if
+	// this DB cluster is created as a read replica.
+	ReplicationSourceIdentifier *string `json:"replicationSourceIdentifier,omitempty"`
+	// For DB clusters in serverless DB engine mode, the scaling properties of the
+	// DB cluster.
+	ScalingConfiguration *ScalingConfiguration `json:"scalingConfiguration,omitempty"`
+	// SourceRegion is the source region where the resource exists. This is not
+	// sent over the wire and is only used for presigning. This value should always
+	// have the same region as the source ARN.
+	SourceRegion *string `json:"sourceRegion,omitempty"`
+	// A value that indicates whether the DB cluster is encrypted.
+	StorageEncrypted *bool `json:"storageEncrypted,omitempty"`
+	// Tags to assign to the DB cluster.
+	Tags []*Tag `json:"tags,omitempty"`
```

Comment:
Do we not strip tags from spec?

Reply:
@surajkota see above.

Reply:
I am assuming its something related to tags, but what should I be looking at?

Reply:
@surajkota did we remove the Tags filtering in order to get SageMaker to preview?

Reply:
@surajkota @RedbackThomson ignore me. We have not been stripping tags from spec.

Reply:
For SageMaker we did ignore tag for preview